tests/components/template/test_image.py

```diff
@@ -582,3 +586,20 @@ async def test_device_id(
     template_entity = entity_registry.async_get("image.my_template")
     assert template_entity is not None
     assert template_entity.device_id == device_entry.id
+
+
+@pytest.mark.freeze_time("2024-07-09 00:00:00+00:00")
+async def test_flow_preview(
+    hass: HomeAssistant,
+    hass_ws_client: WebSocketGenerator,
+) -> None:
+    """Test the config flow preview."""
+
+    state = await async_get_flow_preview_state(
+        hass,
+        hass_ws_client,
+        image.DOMAIN,
+        {"name": "My template", "url": "http://example.com"},
+    )
+
+    assert state["state"] == "2024-07-09T00:00:00+00:00"
```

Comment:
should this test also assert the image bytes?

Reply:
Hum what should be on the preview then? I just tested it out, and the preview looks confusing:

<img width="565" height="663" alt="image" src="https://github.com/user-attachments/assets/34a0a2bc-9c32-4f8b-b2f1-12d8d0990d85" />

Reply:
Yep, the entity inside an entity row doesn't really show much.

Reply:
as far as I can tell, I can't change the preview card type.